applications/nrf_desktop/configuration/nrf52840dk_nrf52840/prj.conf

```diff
@@ -57,6 +57,7 @@ CONFIG_DESKTOP_BLE_SECURITY_FAIL_TIMEOUT_S=10
 CONFIG_DESKTOP_BLE_LOW_LATENCY_LOCK=y
 
 CONFIG_DESKTOP_CONFIG_CHANNEL_ENABLE=y
+CONFIG_DESKTOP_CONFIG_CHANNEL_DFU_ENABLE=y
```

Comment:
The static partition map names must be updated. Otherwise we risk that we use dynamic partition layout which is not good when we support DFU (we must ensure consistent partition layout for DFU)